Description: Debian browser config.
 On Debian systems we don't have to guess the browser.
--- a/src/misc_tcl/webhelp.tcl
+++ b/src/misc_tcl/webhelp.tcl
@@ -16,13 +16,10 @@
 
     } else {
 
-	foreach executable {firefox mozilla netscape iexplorer opera konquerer galeon amaya mosaic lynx w3m links browsex elinks} {
-	    set executable [auto_execok $executable]
-	    if [string length $executable] {
-		catch { exec sh -c "$executable -remote \"openURL(${prefix}//${file},new-window)\" || $executable ${prefix}//${file} " 2> /dev/null & }
-		break
-	    }
-	}
+## There is no need for trial-and-error search on a Debian system
+## -> there is 'sensible-browser'
+
+	catch { exec sensible-browser ${prefix}//${file} 2> /dev/null & }
     }
 }
 
